modules/core/04-channel/migrations/v11/store.go

```diff
@@ -0,0 +1,70 @@
+package v11
+
+import (
+	"fmt"
+
+	corestore "cosmossdk.io/core/store"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/cosmos/ibc-go/v10/modules/core/04-channel/types"
+	"github.com/cosmos/ibc-go/v10/modules/core/keeper"
+)
+
+const (
+	KeyNextSeqSendPrefix = "nextSequenceSend"
+	KeyChannelEndPrefix  = "channelEnds"
+	KeyChannelPrefix     = "channels"
+	KeyPortPrefix        = "ports"
+)
+
+// NextSequenceSendV1Key returns the store key for the send sequence of a particular
+// channel binded to a specific port.
+func NextSequenceSendKey(portID, channelID string) []byte {
```

Comment:
Should this be named like the comment suggests?
```suggestion
func NextSequenceSendV1Key(portID, channelID string) []byte {
```